.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ exclude: >
         \.map$|
         \.map\.js$|
         ^tests/sentry/lang/.*/fixtures/|
```

Comment:
This path matches nothing anymore, can remove.

Reply:
sweet, will do in a followup!